pennylane/fourier/qnode_spectrum.py

```diff
@@ -311,21 +375,21 @@ def circuit(x, y, z):
                 return qml.expval(qml.PauliZ(wires=0))
 
         First, note that we assigned ``id``s to the gates for which we will use
```

Comment:
```suggestion
        First, note that we assigned ``id`` labels to the gates for which we will use
```
Sphinx doesn't seem to like alphanumeric characters directly beside code blocks:
![image](https://user-images.githubusercontent.com/2959003/136151329-5e8f018b-2245-4891-aa8d-5ed6f5d173c5.png)